R/getMadratMessage.R

```diff
@@ -0,0 +1,31 @@
+#' Tool: getMadratMessage
```

Comment:
This makes me think it is a function with the "tool" prefix, but it is not. So maybe delete the "Tool: " here?
```suggestion
#' getMadratMessage
```